apps/Unciv/install-32

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+### make directory ~/Unciv if not available
+if [[ ! -d ~/Unciv ]]; then
+  mkdir ~/Unciv || error 'Failed to make folder $HOME/Unciv!'
+fi
+
+### Add LICENSE from Github if doesn't exist
+if [[ ! -f ~/Unciv/LICENSE ]]; then
+  wget -O ~/Unciv/LICENSE ~ https://raw.githubusercontent.com/yairm210/Unciv/master/LICENSE || error 'Failed to make folder $HOME/Unciv!'
```

Comment:
The error message on this line does not make sense. Also, why is there a `~` character in this wget command?

Reply:
Unintended. Fixed in https://github.com/touhidurrr/pi-apps/commit/ff9b43b14eb696080bd03d096f4f9c2d43f42236.